app/controllers/application_controller.rb

```diff
@@ -9,6 +9,11 @@ class ServiceDisabled < StandardError; end
     render 'errors/service_disabled'
   end
 
+  rescue_from JsonApi::ApiError do |e|
+    logger.error "[Handled] #{e.class}: #{e.message}"
```

Comment:
Really like the `[Handled]` prefix here :+1:

Reply:
Thanks :) 

Just for interest: there's actually support for 'tagged' errors, but this didn't seem like the intended use case for them: http://api.rubyonrails.org/classes/ActiveSupport/TaggedLogging.html#method-i-tagged